change-validation/src/main/scala/com/normation/plugins/changevalidation/WorkflowService.scala

```diff
@@ -435,6 +441,15 @@ class TwoValidationStepsWorkflowServiceImpl(
     toFailure(Deployment, changeRequestId, actor, reason)
   }
 
+  private[this] def stepDeploymentToDeployed(
```

Comment:
This was removed in 7.3 by https://github.com/Normation/rudder-plugins/commit/3103cf4338d98ed30425e4fbb5e31ba449433141#diff-98e8dc3343e38546c42491a39303758ad01214ed8e5ef40c6a2c4d85cfa73f8eL413

This is a missing steps, don't understand why it's have been removed